src/main/java/com/example/Mini/product/productDTO/requset/CreateProductRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.example.Mini.product.productDTO.requset;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.web.multipart.MultipartFile;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class CreateProductRequest {
+    private String productName;
```

Comment:
```suggestion
    private String name;
```
(sửa cả các chỗ khác)